.github/actions/generate-docker-tag/action.yml

```diff
@@ -24,8 +24,8 @@ runs:
     - name: Determine Full Docker Image Tag
       shell: bash
       run: |
-        echo "TT_METAL_DOCKER_IMAGE_TAG=ghcr.io/${{ github.repository }}/tt-metalium/${{ inputs.image }}:${{ env.IMAGE_TAG }}" >> $GITHUB_ENV
-        echo "TT_METAL_REF_IMAGE_TAG=ghcr.io/${{ github.repository }}/tt-metalium/${{ inputs.image }}:latest" >> $GITHUB_ENV
```

Comment:
This does not allow us to use images that follow the convention outlined in:
https://github.com/tenstorrent/tt-metal/issues/12495
i.e.
`tt-metalium-{OS}-{ARCH}-release/grayskull`